src/background/installer.ts

```diff
@@ -21,6 +21,7 @@ import urljoin from "url-join";
 import { reportEvent, initTelemetry } from "@/telemetry/events";
 import { DNT_STORAGE_KEY, getDNT, getUID } from "@/background/telemetry";
 
+// eslint-disable-next-line prefer-destructuring -- It breaks EnvironmentPlugin
 const SERVICE_URL = process.env.SERVICE_URL;
```

Comment:
We could avoid these exception but just not using uppercase constants, just like any other "`const` variable" defined in the code:

```suggestion
const serviceUrl = process.env.SERVICE_URL;
```